config/taxonomy/Makefile

```diff
@@ -1,10 +1,27 @@
 ROOT_DIR := ../..
-all: compile-taxonomy-schema validate
+include $(ROOT_DIR)/Makefile.env
+include $(ROOT_DIR)/hack/make-rules/tools.mk
+
+.DEFAULT_GOAL := all
+
+all: clean compile-taxonomy-schema validate model-codegen
 
 compile-taxonomy-schema:
 	go run $(ROOT_DIR)/main.go taxonomy compile --base ./base/base.yaml --out ./objects/taxonomy.json \
-		--codegen  
+		--codegen
 
 validate:
 	go run ../../main.go taxonomy validate ./objects/policy_manager_request.json
-	go run ../../main.go taxonomy validate ./objects/policy_manager_response.json
\ No newline at end of file
+	go run ../../main.go taxonomy validate ./objects/policy_manager_response.json
+
+model-codegen: $(TOOLBIN)/openapi-generator-cli
```

Comment:
Consider making this a stand alone target so it would generate the json -> generate code -> remove json.

Reply:
Renamed the target to `compile-codegen-cleanup`. Please review. Thanks.